real-main/app/models/user/dynamo/base.py

```diff
@@ -168,6 +173,16 @@ def process_attr(name, value):
                 else:
                     expression_actions['REMOVE'].append(name)
 
+        if current_location is not None:
+            latitude = BasicContext.create_decimal(current_location['latitude']).normalize()
+            longitude = BasicContext.create_decimal(current_location['longitude']).normalize()
+            accuracy = current_location.get('accuracy')
+
+            if accuracy is not None:
+                accuracy = BasicContext.create_decimal(accuracy).normalize()
+
+            current_location = {"latitude": latitude, "longitude": longitude, "accuracy": accuracy}
```

Comment:
This is better - but this will still sometimes try to write null/None into dynamo...

```suggestion
            current_location = {"latitude": latitude, "longitude": longitude}
            if 'accuracy' in current_location:
                current_location['accuracy'] = BasicContext.create_decimal(current_location['accuracy']).normalize()            
```

Please make sure the unit tests for this function check the situations where
- accuracy is not set
- accuracy is set